hg-fast-export.py

```diff
@@ -161,7 +161,15 @@ def export_file_contents(ctx,manifest,files,hgtags,repourl,revnode,ignoreSub,enc
     if not hgtags and file == ".hgtags":
       sys.stderr.write('Skip %s\n' % (file))
       continue
+    # Skip .git files. They only get us in trouble.
+    if '.git' in file:
+      sys.stderr.write('Skip %s\n' % (file))
+      continue
     d=ctx.filectx(file).data()
+    # Skip files bigger than 3 MB
+    if len(d) > 3145728:
```

Comment:
Ok, they should not be versioned, but for some cases its needed.
And removing all files lager than 3MB by default is not a good idea.
But you can add a parameter to enable it if someone want to remove it...